src/libs/actions/PersonalDetails.js

```diff
@@ -81,6 +82,24 @@ function extractFirstAndLastNameFromAvailableDetails({
     };
 }
 
+/**
+ * Convert country names to their respective ISO codes obtained from the backend
+ * This is for backward compatibility of stored data before E/App#15507
+ * @param {String} countryName
+ * @returns {String}
+ */
+function getCountryISO(countryName) {
+    if (countryName === '' || countryName.length === 2) {
+        return countryName;
+    }
+    const isoCodes = _.keys(CONST.ALL_COUNTRIES);
+    for (let i = 0; i < isoCodes.length; i++) {
+        if (CONST.ALL_COUNTRIES[isoCodes[i]] === countryName) {
+            return isoCodes[i];
+        }
+    }
```

Comment:
Can't you simplify this with underscore like so?

Note: the `|| '';` at the end is to make sure we do actually return a string always

```suggestion
    return _.find(CONST.ALL_COUNTRIES, (country) => {
        return country === countryName;
    }) || '';
```